Update docs on authenticator friendly name change and authenticationOptions (#3428)
wso2/docs-is: en/docs/deploy/migrate/what-has-changed.md

    The i18n keys of both OTP authenticators have been moved to the i18n property file of the authentication portal.

### Authenticator Display Name

> Display names (Friendly names) of some of the local authenticators and handlers were renamed from Identity server 6.0.0 onwards. The following table maps the previous authenticator display names to the newer ones.

Reviewer: ```suggestion
From IS 6.0.0 onward the display names of some of the local authenticators and handlers were renamed. The following table maps the previous authenticator display names to the IS 6.0.0 display names.
```

<table>
    <tr>
        <th>Authenticator</th>
        <th>Previous display name</th>
        <th>Current display name</th>
    </tr>
    <tr>

>         <td>BasicAuthRequestPathAuthenticator</td><td>basic-auth</td><td>Basic Auth</td>

Reviewer: ```suggestion
        <td><code>BasicAuthRequestPathAuthenticator</code></td><td>basic-auth</td><td>Basic Auth</td>
```

    </tr>
    <tr>

>         <td>IdentifierExecutor</td><td>identifier-first</td><td>Identifier First</td>

Reviewer: ```suggestion
        <td><code>IdentifierExecutor</td><td>identifier-first</code></td><td>Identifier First</td>
```

    </tr>
    <tr>

>         <td>JWTBasicAuthenticator</td><td>jwt-basic</td><td>JWT Basic</td>

Reviewer: ```suggestion
        <td><code>JWTBasicAuthenticator</code></td><td>jwt-basic</td><td>JWT Basic</td>
```

    </tr>
    <tr>

>         <td>OAuthRequestPathAuthenticator</td><td>oauth-bearer</td><td>OAuth Bearer</td>

Reviewer: ```suggestion
        <td><code>OAuthRequestPathAuthenticator</code></td><td>oauth-bearer</td><td>OAuth Bearer</td>
```

    </tr>
    <tr>

>         <td>FIDOAuthenticator</td><td>fido</td><td>Security Key/Biometrics</td>

Reviewer: ```suggestion
        <td><code>FIDOAuthenticator</code></td><td>fido</td><td>Security Key/Biometrics</td>
```

    </tr>
    <tr>

>         <td>totp</td><td>totp</td><td>TOTP</td>

Reviewer: ```suggestion
        <td><code>totp</code></td><td>totp</td><td>TOTP</td>
```

    </tr>
    <tr>

>         <td>BasicAuthenticator</td><td>basic</td><td>Username & Password</td>

Reviewer: ```suggestion
        <td><code>BasicAuthenticator</code></td><td>basic</td><td>Username & Password</td>
```

    </tr>
    <tr>

>         <td>x509CertificateAuthenticator</td><td>X509Certificate</td><td>X509 Certificate</td>

Reviewer: ```suggestion
        <td><code>x509CertificateAuthenticator</code></td><td>X509Certificate</td><td>X509 Certificate</td>
```

    </tr>
    <tr>

>         <td>SessionExecutor</td><td>active-sessions-limit-handler</td><td>Active Sessions Limit</td>

Reviewer: ```suggestion
        <td><code>SessionExecutor</code></td><td>active-sessions-limit-handler</td><td>Active Sessions Limit</td>
```
